plugins/obs-webrtc/whip-output.cpp

```diff
@@ -62,6 +65,10 @@ bool WHIPOutput::Start()
 
 	is_av1 = (strcmp("av1", obs_encoder_get_codec(encoder)) == 0);
 
+	obs_data_t *video_settings = obs_encoder_get_settings(encoder);
+	video_bitrate = (int)obs_data_get_int(video_settings, "bitrate");
```

Comment:
This only works for CBR and similar modes. Is there any consideration made for if a user uses an encoder in CQP or similar modes? Additionally, I could see it being an issue if an encoder has a "bitrate" value applied to its settings object, but the encoder is not using a mode which actually references the "bitrate" value.

Reply:
Also OBS does allow changing the bitrate while streaming, currently this PR does not adapt to that but only reads the bitrate once on stream start so this is not optimal as is
and I'm not sure how well this plays with VBR